Portage/GHCCore.hs

```diff
@@ -144,6 +148,40 @@ ghc741 = (ghc [7,4,1], mkIndex ghc741_pkgs)
 -- Source: http://haskell.org/haskellwiki/Libraries_released_with_GHC
 --         and our binary tarballs (package.conf.d.initial subdir)
 
+
+ghc821_pkgs :: [Cabal.PackageIdentifier]
+ghc821_pkgs =
+  [ p "array" [0,5,2,0]
+  , p "base" [4,10,0,0]
+  , p "binary" [0,8,5,1] -- used by libghc
+  , p "bytestring" [0,10,8,2]
+--  , p "Cabal" [2,0,0,0]  package is upgradeable
+  , p "containers" [0,5,10,2]
+  , p "deepseq" [1,4,3,0] -- used by time
+  , p "directory" [1,3,0,2]
+  , p "filepath" [1,4,1,2]
+  , p "ghc-boot" [8,2,1]
+--  , p "ghc-boot-th" [] err... vOv
+  , p "ghc-compact" [0,1,0,0] -- new addition
+  , p "ghc-prim" [0,5,1,0]
+  , p "ghci" [8,2,1]
+  , p "hoopl" [3,10,2,2] -- used by libghc
+  , p "hpc" [0,6,0,3] -- used by libghc
+  , p "integer-gmp" [1,0,0,1]
+--  , p "pretty" [] the copy in ghc is used only by ghc
```

Comment:
pretty is still a separate package from cabal packaging standpoint. It should not be dropped from list.
`ghci -package=pretty` is usable just fine.

```
$ ghc-pkg list pretty
/usr/lib64/ghc-8.2.1/package.conf.d
    pretty-1.1.3.3
```